### PR TITLE
reside-32: Add draft dde post

### PR DESCRIPTION
**Pull request by richfitz**
Apologies for the rapid fire, but I thought that this would get held up longer at CRAN.  We should delay this one until Monday (which is when it is dated for) because it takes a few days for binaries to be built

### CODE REVIEW

File: content/blog/dde-1.0.0.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+author: "Rich FitzJohn"
+date: 2019-05-20
+title: dde 1.0.0
+best: false
+---
+
+Last week we released the first public version of [`dde`](https://mrc-ide.github.io/dde/).  This package implements a method for solving [delay differential equations](https://en.wikipedia.org/wiki/Delay_differential_equation), which we use with [`odin`](https://mrc-ide.github.io/odin) to model disease dynamics.
+
+With ordinary differential equations, you express the system of equations as *dy/dt = f(y(t), t)*; the rate of change of the system depends on the current state of the system and the current time, but with delay differential equations *dy/dt* also depends on *y(t - tau)*.  In general these are hard to solve numerically but there is a large class of useful equations with constant delays that are both interesting and tractable.
```
Comment (hillalex) [inline content/blog/dde-1.0.0.md]: Where `tau` is an earlier point in time? Extremely minor point but I first read this as `t*a*u` and was like, what are `a` and `u`, `τ` would be clearer!

**Review by hillalex — COMMENTED** [review]
Looks good! Minor comment

File: content/blog/dde-1.0.0.md (hunk 1 above)
Comment (richfitz) [reply content/blog/dde-1.0.0.md]: Thanks - this is much better

**Review by weshinsley — APPROVED** [review]
Looks fine to me - no suggestions I can think of.

**Pull request merged**